content/how-to-guides/how-to-handle-errors.md

```diff
@@ -17,13 +17,13 @@ The Stacks 2.0 Blockchain API is centrally hosted. However, every running Stacks
 
 While the Node RPC API doe not provide the same functionality as the hosted Stacks 2.0 Blockchain API, you will have similar functionality in a way that is scoped to that specific node. The RPC API includes the following endpoints:
```

Comment:
```suggestion
While the Stacks Node RPC API does not provide the same functionality as the hosted Stacks Blockchain API, you will have similar functionality in a way that is scoped to that specific node. The RPC API includes the following endpoints:
```

Reply:
Aside from the nit and a typo, I do not follow this statement. Suppose the goal is to say Hiro's Stacks Blockchain API supports the following endpoints that act as a proxy to fetch the raw Stacks RPC endpoint responses. In that case, the latter part of the statement (i.e., "The RPC API includes the following endpoints:") is confusing. Perhaps we could rephrase.

Reply:
it prob makes sense just to remove that last line, will go ahead and do that